src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/DateTimeConverter.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI
+{
+    /// <summary>
+    /// DateTime Converter for published date in package metadata.
+    /// </summary>
+    public class DateTimeConverter : IValueConverter
+    {
+        /// <summary>
+        /// Converts DateTime object into custom formatting.
+        /// Goal of the converter is to have custom formatting for japanese since "D" formatting does not include day of the week.
+        /// </summary>
+        /// <returns>string value of the date time.</returns>
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value == null)
+            {
+                return null;
+            }
+
+            var dateTime = DateTime.Parse(value.ToString());
+            if (string.Equals(culture.Name, "ja-JP"))
```

Comment:
Nit: can you add a comment explaining why this is special cased

Reply:
https://github.com/NuGet/NuGet.Client/blob/f3c67bcb62e8b3ab784a038855a67138aa42174b/src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/DateTimeConverter.cs#L17